starport/services/network/networktypes/campaign.go

```diff
@@ -31,3 +32,38 @@ func ToCampaign(campaign campaigntypes.Campaign) Campaign {
 		TotalShares:        campaign.TotalShares.String(),
 	}
 }
+
+// MainnetAccount represents the campaign mainnet account of a chain on SPN
+type MainnetAccount struct {
+	Address string `json:"Address"`
+	Shares  string `json:"Shares"`
```

Comment:
I think we still should use sdk.Coins here, no need to limit it a string.

Reply:
We should convert to string here or when we print the summary. Any place works and is clean

Reply:
I kind of prefer sdk.Coins because I'm also taking programmatical use of service/network package. It can be imported by another repo by someone else. Cmd is and print this info there is won't be the only use case I think.